clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -38,8 +38,13 @@ include "mlir/IR/CommonAttrConstraints.td"
 // CIR Ops
 //===----------------------------------------------------------------------===//
 
+class LoweringInfo {
+  bit simpleLowering = 0;
+  string llvmOpName;
```

Comment:
How about we just have a single `string simpleLoweringLLVMOpName = ""` variable, and if it's defined to anything other than empty it implies simple lowering?

Reply:
Yeah I'm thinking about exactly the same thing... Updated.